.babelrc

```diff
@@ -1,5 +1,9 @@
 {
-  "presets": ["es2015", "stage-0", "react"],
+	"presets": [
+    ["env", {"targets": { "electron": 1.4 }, "useBuiltIns": true }],
```

Comment:
I think we should use `"node": 6` target, we need use for `build`, `test` and `hot-server`, make sure that node and electron will be different in the future, although currently they are the same. @amilajack What do you think?